dropwizard-jetty/src/main/java/io/dropwizard/jetty/ContextRoutingHandler.java

```diff
@@ -54,4 +54,9 @@ protected void doStop() throws Exception {
             handlers.get(key).stop();
         }
     }
+
+    @Override
+    public Handler[] getHandlers() {
+        return handlers.keySet().stream().map(key -> handlers.get(key)).toArray(Handler[]::new);
```

Comment:
Changing `Trie` to `ArrayTernaryTrie` introduces `entrySet` which will eliminate the lookup in a loop.

```java
private final ArrayTernaryTrie<Handler> handlers;
handlers.entrySet().stream().map(e -> e.getValue()).toArray(Handler[]::new);
```

https://www.eclipse.org/jetty/javadoc/9.4.11.v20180605/org/eclipse/jetty/util/ArrayTernaryTrie.html#entrySet--